WordPress/src/main/java/org/wordpress/android/ui/sitecreation/segments/NewSiteCreationSegmentViewHolder.kt

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.sitecreation.segments
+
+import android.support.annotation.LayoutRes
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import org.wordpress.android.R
+import org.wordpress.android.fluxc.model.vertical.VerticalSegmentModel
+import org.wordpress.android.util.StringUtils
+import org.wordpress.android.util.image.ImageManager
+import org.wordpress.android.util.image.ImageType.IMAGE
+
+sealed class NewSiteCreationSegmentViewHolder(internal val parent: ViewGroup, @LayoutRes layout: Int) :
```

Comment:
I assume you're planning to add more view holders that's why you have a `sealed class`. Is that right?

Reply:
I'm not really planning to add more, but I feel like this is a good pattern and it's easily open for extension when we need to add more types, yet it doesn't make things complex. So it felt like a "good deal".

But tbh maybe we should add even the header into the recycler view so the whole screen is scrollable - wdyt?

Reply:
Maybe it's not such a good idea to put the header into the recycler view as we want to be able to show the header + progress bar -> we'd have to create another ListItemType for progress, but it'd a bit complicated to make it match the rest of the screen.

So question is whether you think I should remove the sealed class when I don't plan to create another item type at the moment.

Reply:
> So question is whether you think I should remove the sealed class when I don't plan to create another item type at the moment

I think that's best for now. It's easy enough to refactor when we need it.